<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <arg name="node_name" />

  <node pkg="lights_control" type="controller" name="$(arg node_name)"/>
  <param name="id" value="10" />
  <param name="name" value="luces_habitacion" />
  <param name="state" value="0" />
</launch>
